custom/icds_reports/models/aggregate.py

```diff
@@ -45,8 +45,11 @@
 )
 
 
-def get_cursor(model):
-    db = db_for_read_write(model)
+def get_cursor(model, force_citus=False):
+    if force_citus:
+        db = CITUS_MASTER
+    else:
+        db = db_for_read_write(model)
```

Comment:
can this not be handled directly in `db_for_read_write`  using a thread local? It seems like that might cut down the number of changes dramatically.